packages/phone-number-privacy/combiner/src/bls/bls-cryptography-client.ts

```diff
@@ -50,7 +53,6 @@ export class BLSCryptographyClient {
       const err = new Error(
         `${ErrorMessage.NOT_ENOUGH_PARTIAL_SIGNATURES} ${this.verifiedSignatures.length}/${threshold}`
       )
-      logger.error({ err })
```

Comment:
Why remove the log here?

Reply:
I can't find an example at the moment. But it was getting logged twice because the error is caught and logged later on.